### ISSUE TITLE
The outputs of the R-CNN ILSVRC13 model is incorrect

### ISSUE THREAD

**Issue by anirudhacharya** [onnx/models#46]
For the onnx model - [R-CNN ILSVRC13](https://github.com/onnx/models/tree/master/bvlc_reference_rcnn_ilsvrc13) The model outputs seem to be incorrect for the task of object detection. The model output has the dimension (1,200) and each of values in the tensors seem to be a negative value around -2.5. But for an object detection task we ideally need information of the 
 - bounding box co-ordinates
 - classes of objects detected in each region/bounding box.

The output we get from the model, the (1,200) shape tensor does not provide this information.

The model README says that it is an implementation of [this paper](https://arxiv.org/pdf/1311.2524.pdf) but the output does not seem to be accomplishing the task of object detection.

Can someone verify this?

**Comment by lupesko** [comment]
@houseroad can you look into it please?

**Comment by houseroad** [comment]
I have checked the details in caffe and our model zoo. This R-CNN ILSVRC13 caffe model is not the full model. Actually, it is the classifier for the proposed region. The region proposal is implemented in python script.

So I think the model is fine, and we have tested it end to end. I will add some description to model description.

Reference:
http://nbviewer.jupyter.org/github/BVLC/caffe/blob/master/examples/detection.ipynb
